app/code/Magento/Catalog/Test/Mftf/Test/AdminCreateProductAttributeRequiredTextFieldTest.xml

```diff
@@ -50,7 +50,7 @@
         <waitForPageLoad stepKey="waitForProductToLoad"/>
 
         <fillField selector="{{AdminProductFormSection.productQuantity}}" userInput="100" stepKey="fillProductQty"/>
-        <selectOption selector="{{AdminProductFormSection.productStockStatus}}" userInput="In Stock" stepKey="selectStockStatus"/>
+        <actionGroup ref="AdminSetStockStatusActionGroup" stepKey="selectStockStatus"/>
```

Comment:
Please, specify values explicitly instead of using the default parameter value of the action group. In that way, we will make the test much more readable (don't need to go and check the default value for the parameter in the action group to figure out what value will be set in this action). 

Thanks.

Reply:
I have addressed this, thank you.